src/css/main.css

```diff
@@ -1,4 +1,85 @@
+
+header {
+    background: #f0f0f0;
+    position: absolute;
+    top: 0;
+    width: 100%;
+}
+
+#header img{
+    position:  absolute;
+    right: 5%;
+    background-color: #f0f0f0;
+    padding: 16px, 16px, 0px, 16px;
+
+}
+
+#header h1 {
+    display: inline;
+    position: left;
+    left: 5%;
+    color: black;
+    position: left;
+    margin: 0%;
+    font-size: 24px;
+    padding: 16px, 16px, 0px, 16px;
+    border: #C0C0C0;
+}
+
+article {
+    padding-top: 150px;
+}
+
 p {
-    color: red;
-    appearance: none;
+    color: black;
+    -webkit-appearance: none;
```

Comment:
so... you are adding the prefix to the css yourself whilst you have a task for this in your postprocess job

Reply:
I think i left that in by accident from when I was testing the task. I will remove it.